addon/components/o-s-s/progress-bar.stories.js

```diff
@@ -0,0 +1,91 @@
+import hbs from 'htmlbars-inline-precompile';
+
+const ProgressBarSkins = ['primary', 'warning', 'success'];
+
+export default {
+  title: 'Components/OSS::ProgressBar',
+  component: 'progress-bar',
+  argTypes: {
+    value: {
+      type: { required: true },
+      description: 'Numbered value between 0 & 100',
+      table: {
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'number' }
+    },
+    label: {
+      description: 'Displayed label',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: 'Hello' }
```

Comment:
```suggestion
        defaultValue: { summary: 'undefined' }
```
The default value is `undefined` here